### PR TITLE
chore: merge the 1.6 branch back into master

### PR DESCRIPTION
**Pull request by docmerlin**
chore: merge the 1.6 branch back into master

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@
 - [#2454](https://github.com/influxdata/kapacitor/pull/2454): Add PrimaryProperty and SecondaryProperty methods to BigPanda AlertNode.
 - [#2462](https://github.com/influxdata/kapacitor/pull/2462): BREAKING: we are forced to remove support for 386/darwin builds as go doesn't support them anymore.
 - [#2475](https://github.com/influxdata/kapacitor/pull/2475): Add default color theme to teams alerts
+- [#2512](https://github.com/influxdata/kapacitor/pull/2512): Pull in auth code from Kapacitor Enterprise
```
Comment (danxmoran) [inline CHANGELOG.md:33]: This looks like the wrong section, should it be moved up?

Comment (danxmoran) [reply CHANGELOG.md:33]: And should lines be added for all the PRs that were merged to `v1.6-master`?

Comment (docmerlin) [reply CHANGELOG.md:33]: Good eye, I'll fix that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,9 @@
 ## Unreleased
 
 ### Features
-- [#2512](https://github.com/influxdata/kapacitor/pull/2512): Pull in auth code from Kapacitor Enterprise
+- [#2484](https://github.com/influxdata/kapacitor/pull/2484): Add Zenoss alert avent handler.
```
Comment (danxmoran) [inline CHANGELOG.md:6]:
```suggestion
- [#2484](https://github.com/influxdata/kapacitor/pull/2484): Add Zenoss alert event handler.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,9 @@
...
+- [#2512](https://github.com/influxdata/kapacitor/pull/2512): Pull in auth code from Kapacitor Enterprise.
+- [#2493](https://github.com/influxdata/kapacitor/pull/2493): Route kafka alerts to partitions by ID, and allow for configuring he hashing strategy.
```
Comment (danxmoran) [inline CHANGELOG.md:8]:
```suggestion
- [#2493](https://github.com/influxdata/kapacitor/pull/2493): Route kafka alerts to partitions by ID, and allow for configuring the hashing strategy.
```

**Review by danxmoran — APPROVED** [review]

**Pull request merged**